tests/functional/aws-node-sdk/test/multipleBackend/put/put.js

```diff
@@ -214,6 +221,7 @@ describe('MultipleBackend put object', function testSuite() {
             it('should encrypt body only if bucket encrypted putting ' +
             'object to AWS',
             done => {
+                const key = `somekey-${Date.now()}`;
```

Comment:
you could also generate the key in a beforeEach like ``` this.currentTest.key=`somekey-${Date.now()}` ```
and use it in all the tests `this.test.key`